subsys/bluetooth/controller/hci_driver.c

```diff
@@ -23,6 +23,11 @@
 #define BLE_CONTROLLER_IRQ_PRIO_LOW  4
 #define BLE_CONTROLLER_IRQ_PRIO_HIGH 0
 
+#if (CONFIG_BLECTRL_SLAVE_COUNT + CONFIG_BLECTRL_MASTER_COUNT) != \
```

Comment:
There is the subtle problem that you could have MAX_CONN=5, SLAVE_COUNT=1 and MASTER_COUNT=4 when you really meant SLAVE_COUNT=4 MASTER_COUNT=1. But I suppose the check is better than nothing 🤷‍♂

Reply:
> There is the subtle problem that you could have MAX_CONN=5, SLAVE_COUNT=1 and MASTER_COUNT=4 when you really meant SLAVE_COUNT=4 MASTER_COUNT=1. But I suppose the check is better than nothing 🤷‍♂

In such case configurations between BLE Host (Zephyr) and Nordic LL are aligned, only the Controller is incorrectly configured...